sdk/core/src/policies/telemetry_policy.rs

```diff
@@ -22,36 +21,34 @@ pub struct TelemetryPolicy {
     header: String,
 }
 
-const EMPTY_VERSION: Version = Version {
-    major: 0,
-    minor: 0,
-    patch: 0,
-    pre: Vec::new(),
-    build: Vec::new(),
-};
-
 impl TelemetryPolicy {
     pub fn new(options: TelemetryOptions) -> Self {
-        let platform_info = format!("({}; {}; {})", version().unwrap_or(EMPTY_VERSION), OS, ARCH);
-        if let Some(application_id) = options.application_id {
-            TelemetryPolicy {
-                header: format!(
-                    "{} azsdk-rust-{}/{} {}",
-                    application_id,
-                    clap::crate_name!(),
-                    clap::crate_version!(),
-                    platform_info
-                ),
-            }
-        } else {
-            TelemetryPolicy {
-                header: format!(
-                    "azsdk-rust-{}/{} {}",
-                    clap::crate_name!(),
-                    clap::crate_version!(),
-                    platform_info
-                ),
-            }
+        let crate_name = env!("CARGO_PKG_NAME");
```

Comment:
We should probably just use `option_env!` in the off-chance someone is using this without cargo. We can default to something like "UNKNOWN"

Reply:
I'm talking with the PM that analyzes our telemetry most to figure out good replacements and will adjust accordingly. For example, "0.0.0" might be a good default for `crate_version` so parsing still works if not expecting any other string.

Reply:
"unknown" is fine for the version, she said.